core/src/test/java/org/apache/calcite/util/SourceTest.java

```diff
@@ -40,7 +42,7 @@
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.hasToString;
+import static org.hamcrest.Matchers.*;
```

Comment:
I think Calcite uses explicit imports rather than .*; this is configurable in intellij settings 

https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports